api2-war/src/main/java/eu/europeana/api2/v2/web/controller/SearchController.java

```diff
@@ -907,49 +954,60 @@ public ModelAndView fieldTripRss(
-                                      gftChannelAttributes.get(reqLanguage + "_description");
-                channel.language = gftChannelAttributes.get("language") == null
-                                   || gftChannelAttributes.get("language").equalsIgnoreCase("") ? "--" : gftChannelAttributes.get("language");
-                channel.link = gftChannelAttributes.get("link") == null
-                               || gftChannelAttributes.get("link").equalsIgnoreCase("") ? "no link defined" : gftChannelAttributes.get("link");
-                channel.image = gftChannelAttributes.get("image") == null
-                                || gftChannelAttributes.get("image").equalsIgnoreCase("") ? null : new FieldTripImage(gftChannelAttributes.get("image"));
+                channel.title = gftChannelAttributes.get(reqLanguage + "_" + TITLE) == null ||
+                                gftChannelAttributes.get(reqLanguage + "_" + TITLE).equalsIgnoreCase("") ? (
+                                        gftChannelAttributes.get(TITLE) == null || gftChannelAttributes.get(TITLE)
+                                                                                                       .equalsIgnoreCase(
+                                                                                                               "") ? "no title defined" : gftChannelAttributes
+                                                .get(TITLE)) : gftChannelAttributes.get(reqLanguage + "_" + TITLE);
+                channel.description = gftChannelAttributes.get(reqLanguage + "_" + DESCRIPTION) == null ||
+                                      gftChannelAttributes.get(reqLanguage + "_" + DESCRIPTION).equalsIgnoreCase("") ? (
+                                              gftChannelAttributes.get(DESCRIPTION) == null ||
+                                              gftChannelAttributes.get(DESCRIPTION)
+                                                                  .equalsIgnoreCase("") ? "no description defined" : gftChannelAttributes
+                                                      .get(DESCRIPTION)) : gftChannelAttributes.get(
+                        reqLanguage + "_" + DESCRIPTION);
+                channel.language = gftChannelAttributes.get(LANGUAGE) == null ||
+                                   gftChannelAttributes.get(LANGUAGE).equalsIgnoreCase("") ? "--" : gftChannelAttributes
+                                           .get(LANGUAGE);
+                channel.link = gftChannelAttributes.get(LINK) == null || gftChannelAttributes.get(LINK)
+                                                                                             .equalsIgnoreCase("") ? "no link defined" : gftChannelAttributes
+                                       .get(LINK);
+                channel.image = gftChannelAttributes.get(IMAGE) == null ||
+                                gftChannelAttributes.get(IMAGE).equalsIgnoreCase("") ? null : new FieldTripImage(
```

Comment:
Indentation was bad before, but I feel the altered indentation only made things worse

Reply:
Agreed. I tried some other formatting settings for binary and ternary operations, I think it looks somewhat better now.